sleap/io/format/nix.py

```diff
@@ -0,0 +1,270 @@
+import os
+import numpy as np
+
+from typing import List
+
+from .adaptor import Adaptor, SleapObjectType
+from .filehandle import FileHandle
+from ..dataset import Labels
+from ..video import Video
```

Comment:
It seems safer/more readable to explicitly state where the imports are coming from (i.e. if we move this bit of code elsewhere later on it'll be less work).

Reply:
I guess, I do not agree with this ;) Within the package I would prefer relative imports. But this your decision so I changed imports accordingly.